lib/matplotlib/widgets.py

```diff
@@ -2009,6 +2008,47 @@ def set_visible(self, visible):
         for artist in self.artists:
             artist.set_visible(visible)
 
+    @property
+    def artists(self):
+        """Tuple of the artists of the selector."""
+        if getattr(self, '_handles_artists', None) is not None:
+            return (self._selection_artist, ) + self._handles_artists
+        else:
+            return (self._selection_artist, )
+
+    def set_props(self, **props):
+        """
+        Set the properties of the selector artist. See the `props` argument
+        in the selector docstring to know which properties are supported.
+        """
+        artist = self._selection_artist
+        alias_map = getattr(artist, '_alias_map', None)
+        if alias_map:
+            props = cbook.normalize_kwargs(props, alias_map)
```

Comment:
I believe `normalize_kwargs` handles all this:
```suggestion
        props = cbook.normalize_kwargs(props, artist)
```